incubator/zookeeper/README.md

```diff
@@ -16,31 +15,58 @@ This chart will do the following:
 
 * Create a fixed size ZooKeeper ensemble using a 
 [StatefulSet](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/).
-* Create a [ConfigMap](http://kubernetes.io/docs/user-guide/configmap/) to provide configuration parameters to the Pods 
-in the ensemble.
-* Create a [PodDisruptionBudget](http://kubernetes.io/docs/admin/disruptions/) so kubectl drain will respect the Quorum 
+* Create a [PodDisruptionBudget](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-disruption-budget/) so kubectl drain will respect the Quorum 
 size of the ensemble.
-* Optionally, apply a [Pod Anti-Affinity](http://kubernetes.io/docs/user-guide/node-selection/) annotation to spread the 
+* Create a [Headless Service](https://kubernetes.io/docs/concepts/services-networking/service/) to control the domain of the ZooKeeper ensemble.
+* Create a Service configured to connect to the avialable ZooKeeper instance on the configured client port.
+* Optionally, apply a [Pod Anti-Affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature) to spread the 
 ZooKeeper ensemble across nodes.
 
 ## Installing the Chart
 
-You can install the chart with the release name `my-release` as below.
+You can install the chart with the release name `my-zk` as below.
 
-```bash
+```console
 $ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
-$ helm install --name my-release incubator/zookeeper
+$ helm install --name my-zk incubator/zookeeper
 ```
 
 If you do not specify a name, helm will select a name for you.
 
+### Installed Components
+
+You can use `kubectl get` to view all of the installed components.
+
+```console{%raw}
+$ kubectl get all -l component=zk-my-zk
+NAME            READY     STATUS              RESTARTS   AGE
+po/zk-my-zk-0   1/1       Running             0          1m
+po/zk-my-zk-1   1/1       Running             0          59s
+po/zk-my-zk-2   0/1       ContainerCreating   0          12s
```

Comment:
It looks like some intermediate state during setup was captured here. Intentional?